pandas/tests/indexes/multi/conftest.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pytest
+from pandas import Index, MultiIndex
+
+
+@pytest.fixture
+def idx():
+    major_axis = Index(['foo', 'bar', 'baz', 'qux'])
+    minor_axis = Index(['one', 'two'])
+
+    major_labels = np.array([0, 0, 1, 2, 3, 3])
+    minor_labels = np.array([0, 1, 0, 1, 0, 1])
+    index_names = ['first', 'second']
+    index = MultiIndex(
+        levels=[major_axis, minor_axis],
+        labels=[major_labels, minor_labels],
+        names=index_names,
+        verify_integrity=False
+    )
+    return index
+
+
+@pytest.fixture
```

Comment:
I see that you are iterating over this in a number of tests. why is that?

Reply:
Okay, done with that, but some tests are not relevant to a the MultiIndex class or would need to be refactored.  I marked those with a TODO.  Those that could refactor, I did.  :sweat_smile:

Reply:
yep that's ok. i expect after your refactor will need to go over this again and clean a bit